web-frontend/src/i18n/en_US.json

```
@@ -93,5 +97,8 @@
   "receive": {
     "account-receive": "Generate a link",
     "account-address": "Account address: "
+  },
+  "send-redirect": {
+    "redirecting": "Hey XX, we're transfering you to the send coin page"
```

Maybe you need to split in two here.

```suggestion
    "redirecting-pre": "Hey "
    "redirecting-pos": ", we're transferring you to the send coin page"
```

Then you can interpolate in your code

---

...or use the official i18n lib ^^

---

We can use the support for interpolation now here!

---

yep it's done

---

Thank you @Pierregvx ! 🙏